src/ray/object_manager/object_store_pool.h

```diff
@@ -0,0 +1,62 @@
+#ifndef RAY_OBJECT_STORE_POOL_H
+#define RAY_OBJECT_STORE_POOL_H
+
+#include <list>
+#include <memory>
+#include <mutex>
+#include <vector>
+
+#include <boost/asio.hpp>
+#include <boost/asio/error.hpp>
+#include <boost/bind.hpp>
+
+#include "plasma/client.h"
```

Comment:
let's explicitly wrap non-alpha order includes with clang-format off/on. That way, when this naming/scoping issue is resolved, it will be easy to find and fix.